template/snippet.js

```diff
@@ -50,6 +50,11 @@
     return function(){
       var args = Array.prototype.slice.call(arguments);
       args.unshift(method);
+      if (window.analytics.initialize) {
```

Comment:
I think `initialize` work as well, but this is cleaner: 
```suggestion
      if (window.analytics.initialized) {
```